### ISSUE TITLE
Skylanders Spyro's Adventure [BLES01272] - Game freezing in multiplayer

### ISSUE THREAD

**Issue by Moisynthe** [RPCS3/rpcs3#13321]
### Quick summary

We were trying to play Skylanders Spyro's Adventure in multiplayer mode but in the first second of the game it just freezed and we could not do anything. I don't know if there is a solution for this but if it exists please give me the key.
![Gam](https://user-images.githubusercontent.com/123894594/215345626-67004e0a-8032-4322-842e-7800b069c52f.PNG)

### Attach a log file

[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/10530444/RPCS3.log.gz)

**Comment by RipleyTom** [comment]
Requires a private server which is not part of rpcs3 scope.

**Comment by Moisynthe** [comment]
It was a local multiplayer on the same computer, why does a private server would be needed ?

**Comment by RipleyTom** [comment]
Yeah sorry the issue wasn't really clear and the game does try to contact a private server. I tried it here and it works, I think it's possibly an issue with the timing of the portal emulator, that game was always very sensible to it.

**Comment by Moisynthe** [comment]
Okay thanks for the answer. Sorry for the non-clarity of my description

**Comment by AniLeo** [comment]
Could be fixed with #13723

**Comment by digant73** [comment]
@Moisynthe is that issue fixed?

**Comment by ScottNash042** [comment]
Please close as the OP hasn't updated this issue in over 3 years
and is a duplicate of #13844
